Common/Orders/OrderEvent.cs

```diff
@@ -33,11 +33,21 @@ public class OrderEvent
         public Symbol Symbol;
 
         /// <summary>
+        /// The date and time of this event (UTC).
+        /// </summary>
+        public DateTime Time;
+
+        /// <summary>
```

Comment:
Agree on the Order.Time, the only trouble there is it might be used in current algorithms, so we'll check to see how many users actually reference it. I just don't want to break existing code if possible. As for using UtcTime = order.Time, that isn't completely correct. order.Time is the time the order was placed in UTC, but orderEvent.UtcTime is the time the fill was produced, so I think we need to accept the time parameter explicitly. My concern is that I want new implementations to be forced to set this value correctly, vs assuming it will be set automatically.